### PR TITLE
5430 other petitioners backend

### PR DESCRIPTION
**Pull request by mrinsin**
Cases now have otherPetitioners

### CODE REVIEW

File: shared/src/business/entities/cases/Case.test.js [hunk 1]
```diff
@@ -168,6 +168,49 @@ describe('Case entity', () => {
     });
   });
 
+  describe('Other Petitioners', () => {
+    it('sets the value of otherPetitioners on the case', () => {
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.test.js:172]: Something I would like to see added to these tests is verifying that invalid otherPetitioners will throw a validation error.

File: shared/src/business/entities/cases/Case.js [hunk 2]
```diff
@@ -531,6 +535,10 @@ Case.VALIDATION_RULES = {
     .boolean()
     .optional()
     .description('Reminder for clerks to review the Order to Show Cause.'),
+  otherPetitioners: joi
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.js:538]: If `otherPetitioners` is undefined, will this validation pass?

Comment (rachelschneiderman) [reply shared/src/business/entities/cases/Case.js:538]: Just tried it, yes!

File: shared/src/business/entities/contacts/ContactFactory.js [hunk 3]
```diff
@@ -279,7 +299,16 @@ ContactFactory.createContacts = ({ contactInfo, isPaper, partyType }) => {
     isPaper,
     partyType,
   });
+  let otherPetitioners = [];
+  if (Array.isArray(contactInfo.otherPetitioners)) {
+    otherPetitioners = otherPetitionersCreation(
+      contactInfo.otherPetitioners || [],
```
Comment (sutt0n) [inline shared/src/business/entities/contacts/ContactFactory.js]: i don't think the empty array `[]` would get reached here

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**